code/modules/events/shuttle_loan.dm

```diff
@@ -132,7 +132,7 @@
 					new /obj/structure/spider/stickyweb(T)
 
 			if(ANTIDOTE_NEEDED)
-				var/virus_type = pick(/datum/disease/beesease, /datum/disease/brainrot, /datum/disease/fluspanish)
+				var/obj/item/weapon/reagent_containers/glass/bottle/virus_type = pick(/obj/item/weapon/reagent_containers/glass/bottle/beesease, /obj/item/weapon/reagent_containers/glass/bottle/brainrot, /obj/item/weapon/reagent_containers/glass/bottle/fluspanish)
```

Comment:
I think a dead infected assistant would probably be the best thing; having smoke wouldn't really be that impressive since people would just avoid it.

If it's an assistant that's infected though, then people have to engage in proper containment procedures as if it were blood.